README.md

```diff
@@ -145,107 +125,128 @@ public class Student {
 
-				// access transaction object
+        // access transaction object
         EntityTransaction transaction = entityManager.getTransaction();
 
-				// create and use transactions
+        // create and use transactions
         transaction.begin();
         entityManager.persist(student1);
         transaction.commit();
+
+        //close entity manager and factory
+        entityManager.close();
+        entityManagerFactory.close();
     }
 }
 ```
 
-In the “create a new student instance”, we are creating a new student instance
-which is a regular Java object.
-
-We need to use an `EntityManagerFactory` because we want a single instance of an
-`EntityManager` in our app. We also have to get the `EntityTransaction` object
-from the `EntityManager` because we have to define our own transactions. This is
-boilerplate code required when working with JPA but these aren’t required when
-using frameworks which provide auto configurations such as Spring (with
-SpringBoot).
-
-The `persist` method tells the database to create and insert a student row using
-the data provided.
-
-Make sure your H2 database is running in the terminal or command line and then
-run the `main` method. This will create a `STUDENT_DATA` table in the H2
-database along with a single row of data. In IntelliJ, you can check out the
-“Run” tab to see the exact query that Hibernate used to create the table and
-insert the data. It shows the SQL queries because we enabled this behavior in
-the `persistence.xml` file.
-
-```
-Hibernate:
-
-    drop table if exists STUDENT_DATA CASCADE
-Hibernate:
-
+1. Create a `Student` which is a regular Java object.
+2. Create an `EntityManagerFactory` because we want a single instance of an
+   `EntityManager` in our app. The parameter to `createEntityManagerFactory` should
+   match the string set in the persistence unit  `<persistence-unit name="example" transaction-type="RESOURCE_LOCAL">`.
+3. Get an `EntityTransaction` object from the `EntityManager` because we have
+   to define our own transactions to persist the student in the database.
+4. The `persist` method tells the database to create and insert a table row using
+   the data provided by the `Student` class instance.
+5. Commit the transaction to ensure the row is inserted in the table.
+6. Close the entity manager and factory.
+
+This is boilerplate code required when working with JPA, but this code is not needed when
+using frameworks that provide auto configurations such as Spring (with SpringBoot).
+
+### Run `JpaCreateStudent.main`
+
+1. Run the `JpaCreateStudent.main` method. This will create a `STUDENT_DATA`
+   table in the PostgreSQL `student_db` database and add a new row. 
+2. In IntelliJ, check out the “Run” tab to see the exact query that Hibernate used
+   to create the table and insert the data. It shows the SQL queries because we enabled this behavior in
+   the `persistence.xml` file.
+
+```text
+Hibernate: 
+    
     create table STUDENT_DATA (
```

Comment:
When I look at the table that is created in pgAdmin, it looks like the table name is all lowercase. Does casing not matter then in Postgres?